src/routes/intro/ProfessionSelectionScreen.tsx

```diff
@@ -0,0 +1,121 @@
+import useReadSelectedProfessions from '../../hooks/useReadSelectedProfessions'
+import { RoutesParams } from '../../navigation/NavigationTypes'
+import AsyncStorage from '../../services/AsyncStorage'
+import { childrenDescription } from '../../services/helpers'
+
+const Root = styled.View`
+  background-color: ${props => props.theme.colors.background};
+  height: 100%;
+`
+
+const StyledList = styled(FlatList)`
+  width: 100%;
+` as ComponentType as new () => FlatList<Discipline>
+
+const ButtonContainer = styled.View`
+  padding-top: ${props => props.theme.spacings.md}
+  margin: 0 auto 0;
+`
+const IconContainer = styled.View`
+  margin-right: ${props => props.theme.spacings.sm};
+`
+const Placeholder = styled.View`
+  width: 24px;
+  margin-right: ${props => props.theme.spacings.sm};
+`
+
+interface ProfessionSelectionScreenProps {
+  route: RouteProp<RoutesParams, 'ProfessionSelection'>
+  navigation: StackNavigationProp<RoutesParams, 'ProfessionSelection'>
+}
+
+const ProfessionSelectionScreen = ({ route, navigation }: ProfessionSelectionScreenProps): JSX.Element => {
+  const { discipline } = route.params
+  const { data: disciplines, error, loading, refresh } = useLoadDisciplines(discipline)
+  const { data: selectedProfessions, refresh: refreshSelectedProfessions } = useReadSelectedProfessions()
+
+  const selectDiscipline = async (selectedItem: Discipline): Promise<void> => {
+    if (selectedProfessions?.includes(selectedItem)) {
+      await AsyncStorage.removeSelectedProfession(selectedItem).then(refreshSelectedProfessions)
+    } else {
+      await AsyncStorage.pushSelectedProfession(selectedItem).then(refreshSelectedProfessions)
+    }
+  }
+
+  const Item = ({ item }: { item: Discipline }): JSX.Element => (
+    <DisciplineListItem
+      item={item}
+      onPress={() => selectDiscipline(item)}
+      hasBadge
+      rightChildren={
+        selectedProfessions?.includes(item) ? (
+          <IconContainer>
+            <CheckCircleIconGreen testID='check-icon' />
+          </IconContainer>
+        ) : (
+          <Placeholder />
+        )
+      }
+    />
+  )
+
+  const navigateToHomeScreen = () => {
+    navigation.push('Home')
+  }
+
+  return (
+    <Root>
+      <StatusBar backgroundColor='blue' barStyle='dark-content' />
+      <ServerResponseHandler error={error} loading={loading} refresh={refresh}>
+        <StyledList
+          contentContainerStyle={{ flexGrow: 1 }}
+          ListHeaderComponent={<Title title={discipline.title} description={childrenDescription(discipline)} />}
+          ListFooterComponent={
+            !selectedProfessions || selectedProfessions.length === 0 ? (
+              <ButtonContainer>
+                <Button
+                  onPress={navigateToHomeScreen}
+                  label={labels.intro.skipSelection}
+                  buttonTheme={BUTTONS_THEME.contained}
+                />
+              </ButtonContainer>
+            ) : (
+              <></>
+            )
```

Comment:
I think it would be more intuitive (and easier to implement) if the skip button would be implemented the same way as the confirm selection button. Otherwise the skip button is mostly not visible and the user has to scroll down first. In combination with a very unobtrusive/small text that the user should select their professions I think it is kind of hard to grasp what the user has to do.

Reply:
It was on purpose the "skip" button is kind of "hidden", so the users get encouraged to select a profession. Would keep it for now like this.

Reply:
Okay, then LGTM.